index.html

```diff
@@ -1,224 +1,299 @@
-            <div class="img-four remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-five remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-six remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-seven remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
+          </div>
         </div>
+      </div>
+    </section>
+    <section class="card-section">
+      <div class="cards">
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
```

Comment:
- [ ] Great job 👏 adding the proper button with the stylings but the "See project" button is only supposed to appear when the project is hovered on with the mouse, the description and dark opacity will then disappear and the button appears at the bottom of the image. So please carefully think it over and fix it.
- [ ] After you've implemented it please make sure the button fills the entire project card as shown in the design.